src/App.js

```diff
@@ -1,23 +1,38 @@
-import logo from './logo.svg';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Outlet } from 'react-router-dom';
 import './App.css';
+import { fetchUser } from './features/user/userSlice';
+import { fetchReservatins } from './features/reservations/reservationSlice';
+import SideBar from './components/SideBar';
+import LoginPage from './components/LoginPage';
 
 function App() {
```

Comment:
- [ ] I still can't add cars.  If this is a feature you won't implement, kindly mention that in your PR description, along with the reason (the final evaluator will ask for it, better be prepared now).  👍

Reply:
Also here, due to time restrictions (we have invested too much time in solving code conflicts during the project the remaining time became a big issue to be addressed and caused our delay and not delivering this feature.